big5/operations/ppl.json

```diff
@@ -148,7 +148,7 @@
       "path": "/_plugins/_ppl",
       "method": "POST",
       "body": {
-        "query": "source = {{index_name | default('big5')}} match(`process.name`, 'kernel') | where `@timestamp` >= '2023-01-01 00:00:00' and `@timestamp` < '2023-01-03 00:00:00' | head 10"
+        "query": "source = {{index_name | default('big5')}} {% if distribution_version.split('.') | map('int') | list >= '3.3.0'.split('.') | map('int') | list %}process.name=kernel{% else %}match(`process.name`, 'kernel'){% endif %} | where `@timestamp` >= '2023-01-01 00:00:00' and `@timestamp` < '2023-01-03 00:00:00' | head 10"
```

Comment:
If I understand this correctly, if the version is equal or greater than 3.3.0 then use process.name, else use "\`process.name\`" ? Would this cause existing customer queries to break when they upgrade from lower versions to 3.3.0 and above or is it backward compatible?

Reply:
Yes breaking changes is only in major release, especially API.
I think it is ok to make an API deprecated but still support until next major, but not ok to completely alter its functionalities to the point the original behavior is removed.

cc: @andrross @msfroh @getsaurabh02 into the conversation here, and whether we should restore the original behavior.

Thanks.